crates/re_log/src/setup.rs

```diff
@@ -59,12 +59,21 @@ pub fn setup_native_logging() {
 
 #[cfg(target_arch = "wasm32")]
 pub fn setup_web_logging() {
+    use std::sync::atomic::{AtomicBool, Ordering};
+
+    static LOG_INIT: AtomicBool = AtomicBool::new(false);
```

Comment:
Calling `new WebViewer` multiple times would cause a panic here, because the logger is already set.